scripts/superdesk/keyboard/keyboard.js

```diff
@@ -330,6 +330,37 @@
                 elt['on' + type] = false;
             }
         };
+
+        this.registerShortcut = function (hotkey, callback, options, elem) {
+            this.register(hotkey[0], gettext(hotkey[2]), hotkey[1]);
```

Comment:
Hmmm, if button is visible, then action is available. Not sure if it is good case that we have all shortcuts visible on help screen (available and not available one) because I think it will be confusing. Someone will try to start some action that is visible, but not available, and he will think that it is a bug. So maybe is good case to have only currently available one.

If there is need to display all shortcuts, then I will find some solution for this. But I think we will need to specified them double. And maybe is better to define them only on one place, maybe on this service for displaying them.